### ISSUE TITLE
Pending operations

### ISSUE THREAD

**Issue by thompol** [firecups/ispconfig-firecups-theme#1]
In the DNS module, I see which actions are pending. But nowhere else. Why?
<img width="1166" alt="afbeelding" src="https://user-images.githubusercontent.com/59287450/99184303-2232a000-2742-11eb-8e46-7920020a63f9.png">

**Comment by firecups** [comment]
I will check this, but at now is the "datalog_changes_count" section exist in all relevant files.

**Comment by firecups** [comment]
I have tested on the mail sections. Here i become the pending message too....:

![message](https://user-images.githubusercontent.com/73801258/99187288-0fc26180-2756-11eb-9536-4e2db97e0492.PNG)

**Comment by thompol** [comment]
Yeah, I see now. It is just not on all pages. If implementing it this way, I would like to see it on every page that has a list with domains / accounts.

But as said, I would prefer the "classic" way with a count in the upper right corner.
